alchemist/src/orchid/resources/wiki/usage/maps.md

```diff
@@ -0,0 +1,86 @@
+---
+
+title: Maps and GPS traces
+
+---
+
+Alchemist is equipped with the ability to load and simulate on real-world maps.
+Navigation on maps can be done by using gps traces,
+by moving along roads (Alchemist relies on [GraphHopper](https://www.graphhopper.com/) to provide directions),
+by interpolating gps traces with on-the-road-movements,
+or by ignoring the map information on just move as you would in a continous space.
+
+Configuring a simulation to run on a map, possibly using GPS tracks, requires three parameters to be specified:
+0. Environment
+0. Displacement
+0. Program to follow the GPS traces
+
+## Setting up a map environment
+
+In order to run simulations on real world maps, an appropriate environment must be selected.
+
+Currently, the only environment supporting maps is {{ anchor('OSMEnvironment') }}. 
+
+The only mandatory parameter is the path of the file with the map to be loaded.
+For the optional parameters, refer to the documentation of the environment.
```

Comment:
Aheemmm... nope. That's the documentation. Please expand.

Reply:
done